lib/brakeman.rb

```diff
@@ -180,6 +180,8 @@ def self.get_formats_from_output_format output_format
       [:to_json]
     when :markdown, :to_markdown
       [:to_markdown]
+    when :cc, :codeclimate, :to_codeclimate
```

Comment:
Having `:cc` here doesn't do anything unless it's added to the options parser as well.